temporal-sdk/src/main/java/io/temporal/activity/ActivityOptions.java

```diff
@@ -83,37 +83,57 @@ private Builder(ActivityOptions options) {
     }
 
     /**
-     * Overall timeout workflow is willing to wait for activity to complete. It includes time in a
-     * task queue (use {@link #setScheduleToStartTimeout(Duration)} to limit it) plus activity
-     * execution time (use {@link #setStartToCloseTimeout(Duration)} to limit it). Either this
-     * option or both schedule to start and start to close are required.
+     * Overall timeout workflow is willing to wait for activity to complete.
+     *
+     * <p>ScheduleToCloseTimeout limits total time of the activity execution including retries (use
+     * StartToCloseTimeout to limit a time of a single attempt).
+     *
+     * <p>Either this option or StartToClose are required.
+     *
+     * <p>Defaults to unlimited.
      */
     public Builder setScheduleToCloseTimeout(Duration scheduleToCloseTimeout) {
       this.scheduleToCloseTimeout = scheduleToCloseTimeout;
       return this;
     }
 
     /**
-     * Time activity can stay in task queue before it is picked up by a worker. If schedule to close
-     * is not provided then both this and start to close are required.
+     * Time activity can stay in task queue before it is picked up by a worker.
+     *
+     * <p>Do not specify this timeout unless using host specific task queues for activity task
+     * routing.
+     *
+     * <p>ScheduleToStartTimeout is always non-retryable. Retrying after this timeout doesn't make
+     * sense as it would just put the activity task back into the same task queue.
+     *
+     * <p>Defaults to unlimited.
      */
     public Builder setScheduleToStartTimeout(Duration scheduleToStartTimeout) {
       this.scheduleToStartTimeout = scheduleToStartTimeout;
       return this;
     }
 
     /**
-     * Maximum activity execution time after it was sent to a worker. If schedule to close is not
-     * provided then both this and schedule to start are required.
+     * Maximum time of a single activity execution attempt.
```

Comment:
```suggestion
     * Maximum time of a single Activity execution attempt.
```
We want to capitalize these and treat them as proper nouns.